core/src/com/group66/game/BustaMove.java

```diff
@@ -41,16 +41,8 @@ public int getGameWidth() {
 	 * @see com.badlogic.gdx.ApplicationListener#create()
 	 */
 	@Override
-	public void create () {
+	public void create() {
 		batch = new SpriteBatch();
 		this.setScreen(new MainMenuScreen(this));
 	}
-
-	/* (non-Javadoc)
-	 * @see com.badlogic.gdx.Game#render()
-	 */
-	@Override
```

Comment:
What does this do and what are the implications of removing it?
I'm having a a difficult error in displaying text on screen, would this be somehow related?